packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/AddBankWireForm/Steps/EnterIntermediaryData.tsx

```diff
@@ -76,7 +60,7 @@ const EnterIntermediaryBank = ({ onClickBack, onNextStep }: StepProps) => {
             data-e2e='intermediaryRoutingNumber'
             name='intermediaryRoutingNumber'
             noLastPass
-            validate={[required, validRoutingNumber, onlyNumbers]}
+            validate={[required, onlyNumbers, validRoutingNumber]}
```

Comment:
👏 thank you for making this alphabetical order

Reply:
It really was because `redux-form` run this in order, and I'd first check that this is all numbers and then valid length, but that's a bonus